src/main/java/com/example/lecture09task/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.lecture09task.controller;
+
+import com.example.lecture09task.entity.User;
+import com.example.lecture09task.service.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/users/{id}")
+    public List<User> selectusersid(@PathVariable("id") int id) {
```

Comment:
Javaのメソッドはキャメルケースで書きましょう。
```suggestion
    public List<User> selectUsersId(@PathVariable("id") int id) {
```
またこういうメソッド名にするのもいいと思います。
```suggestion
    public List<User> selectUserById(@PathVariable("id") int id) {
```